examples/clusterautoscaler.yaml

```diff
@@ -25,3 +25,4 @@ spec:
     delayAfterAdd: 10s
     delayAfterDelete: 10s
     delayAfterFailure: 10s
+    unneededTime: 10s
```

Comment:
Not sure this should be the default. Machines will disappear in a production setting far quicker than they should.

Reply:
Ha, this is not supposed to be here. Thanks.